index.md

```diff
@@ -1,5 +1,51 @@
 ---
+title: What is PlaceOS?
 slug: /
+description: Brief intro to the product
 ---
+<!-- This is a placeholder page in markdown to be replaced post-mvp with a more pleasing React page -->
 
-This is a placeholder for the main landing page.
+PlaceOS is a software platform which makes interaction with physical spaces simple.
+It provides tools to design, build and iterate integrated digital experiences for physical environments.
+
+To do this it merges elements on an API gateway, ESB, and traditional BMS, without the legacy baggage.
+This allows disparate systems, devices and services to combine as a seamless UX for your _place_, whatever that is.
+
+## Sections of this site {#sections}
+
+Documentation for PlaceOS fits into one of 4 categories based on the doc's purpose. 
+You can navigate between them at the top of the page.
+
+If you want high-level coverage of topic areas, see the [overview](../overview).
+
+If you want to learn in more detail about a process or feature, try a [tutorial](../tutorial/).
+
+If you're looking for docs to help you with a task, try a [how-to](../how-to/) guide.
+
+For low-level detailed technical docs, see the [reference](../reference/) material.
+
+```text
+                               ▲
+                               │
+                           practical
+                               │
+                               │
+            tutorials          │            how-to
+      (learning-orientated)    │     (problem-orientated)
+                               │
+                               │
+◄─study────────────────────────┼──────────────────────────work─►
+                               │
+                               │
+             explain           │           reference
+   (understanding-orientated)  │   (information-orientated)
+                               │
+                               │
+                          theoretical
+                               │
+                               ▼
+```
+
```

Comment:
Can drop this - this was for the metadocs.

Reply:
I thought some variation on it was useful, but the one-liners for each category above kind of covers it enough.

Reply:
Yeah - was more just in reference to the text diagram. Had only popped that together so that it would display neatly in a MD editor.

A graphic form may be useful, but potentially not a requirement for getting this initial version up. Maybe just swing a comment in there for a TODO?